signer/src/bitcoin/validation.rs

```diff
@@ -32,13 +32,26 @@ use crate::WITHDRAWAL_BLOCKS_EXPIRY;
 use crate::WITHDRAWAL_DUST_LIMIT;
 use crate::WITHDRAWAL_MIN_CONFIRMATIONS;
 
+use super::rpc::assess_mempool_sweep_transaction_fees;
+use super::utxo::compute_transaction_fee;
 use super::utxo::DepositRequest;
+use super::utxo::Fees;
 use super::utxo::RequestRef;
 use super::utxo::Requests;
 use super::utxo::SignatureHash;
 use super::utxo::UnsignedTransaction;
 use super::utxo::WithdrawalRequest;
 
+/// The maximum acceptable multiplier fee difference between the signer and
+/// coordinator estimates when the coordinator's estimate is greater than
+/// the signer's.
+const SIGNER_MAX_BITCOIN_FEE_MULTIPLIER: f64 = 5.0;
```

Comment:
So I remember there being a discussion about config vs. not and landing in that these shouldn't be exposed as config. What's the thinking around having config for Stacks max transaction fee but not these parameters?